packages/fxa-auth-server/lib/routes/sign.js

```diff
@@ -20,30 +20,73 @@ module.exports = (log, signer, db, domain, devices) => {
           strategy: 'sessionToken',
           payload: 'required',
         },
+        description: '🔒 sessionToken',
+        notes: [
+          '<p>🔒 HAWK-authenticated with session token</p>' +
+            `<p>Sign a BrowserID public key. The server is given a public key and returns a signed certificate using the same JWT-like mechanism as a BrowserID primary IdP would (see <a href="https://github.com/mozilla/browserid-certifier"><b>browserid-certifier</b></a> for details). The signed certificate includes a <code>principal.email</code> property to indicate the Firefox Account identifier (a UUID at the account server's primary domain) and is stamped with an expiry time based on the <code>duration</code> parameter.</p>` +
+            '<p>This request will fail unless the primary email address for the account has been verified.</p>' +
+            '<p>Clients should include a query parameter, <code>service</code>, for metrics and validation purposes. The value of <code>service</code> should be <code>sync</code> when connecting to Firefox Sync or the OAuth <code>client_id</code> when connecting to an OAuth relier.</p>' +
+            "<p>If you do not specify a <code>service parameter</code>, or if you specify <code>service=sync</code>, this endpoint assumes the request is from a legacy Sync client. If the session token doesn't have a corresponding device record, one will be created automatically by the server.</p>" +
+            '<p>The signed certificate includes these additional claims:</p>' +
+            '<ul>' +
+            "<li><code>fxa-generation</code>: A number that increases each time the user's password is changed.</li>" +
+            "<li><code>fxa-keysChangedAt</code>: A timestamp that increases each time the user's encryption key is changed.</li>" +
+            "<li><code>fxa-profileChangedAt</code>: A timestamp that increases each time the user's core profile data is changed.</li>" +
+            '<li><code>fxa-lastAuthAt</code>: Authentication time for this session, in seconds since epoch.</li>' +
+            "<li><code>fxa-verifiedEmail</code>: The user's verified recovery email address.</li>" +
+            "<li><code>fxa-tokenVerified</code>: A boolean indicating whether the user's login was verified using an email confirmation or 2FA in addition to their password.</li>" +
+            '<li><code>fxa-amr</code>: A list of strings giving the ways in which the user was authenticated. Possible values include:</li>' +
+            '<ul><li><code>pwd</code>: the user provided the account password</li>' +
+            '<li><code>email</code>: the user completed an email confirmation loop</li>' +
+            '<li><code>otp</code>: the user completed a 2FA challenge</li></ul>' +
```

Comment:
If we remove the trailing `</li>` on L38 above, we should close it here and change this to `</li></ul></li>`.